TASVideos/Pages/Account/Register.cshtml

```diff
@@ -49,6 +49,10 @@
 				</div>
 				<span asp-validation-for="Coppa" class="text-danger"></span>
 			</form-group>
+			<form-group>
+				Also make sure to read the <a href="/SiteRules">Site Rules</a> if you haven't already.<br/>
```

Comment:
I think "if you haven't already" is unnecessary. "Make sure" already implies it.

Reply:
```suggestion
				Also make sure you've read the <a href="/SiteRules">Site Rules</a>.<br/>
```